src/amo/components/UserProfileEdit/index.js

```diff
@@ -143,13 +143,23 @@ export class UserProfileEditBase extends React.Component<Props, State> {
         pictureData: null,
         successMessage: i18n.gettext('Profile successfully updated'),
       });
+      window.scroll({
+       top: 0, 
+       left: 0, 
+       behavior: 'smooth' 
+      });
     }
 
     if (oldUser && oldUser.picture_url && newUser && !newUser.picture_url) {
       this.setState({
         pictureData: null,
         successMessage: i18n.gettext('Picture successfully deleted'),
       });
+      window.scroll({
+       top: 0, 
+       left: 0, 
+       behavior: 'smooth' 
+      });
```

Comment:
For each change you make, we need to cover it with a test case (writing a unit test).